src/tracy-modification.c

```diff
@@ -116,6 +118,40 @@ int tracy_inject_syscall_pre_start(struct tracy_child *child, long syscall_numbe
     child->inj.pre = 1;
     child->inj.syscall_num = syscall_number;
 
+#ifdef __arm__
+    if(child->tracy->opt & TRACY_WORKAROUND_ARM_7475_1) {
+        /* change call to getpid */
+        tracy_modify_syscall_args(child, get_syscall_number_abi("getpid", child->event.abi), &child->inj.reg);
```

Comment:
Gives compiler errors here:

cc -ansi -pedantic -Wall -Wextra -Werror -Wno-unused-result -pipe -Wno-long-long  -D_POSIX_SOURCE -D_GNU_SOURCE -pthread -fPIC -ggdb   -c -o tracy-modification.o tracy-modification.c
tracy-modification.c: In function ‘tracy_inject_syscall_pre_start’:
tracy-modification.c:124:9: error: passing argument 3 of ‘tracy_modify_syscall_args’ from incompatible pointer type [-Werror]
         tracy_modify_syscall_args(child, get_syscall_number_abi("getpid", child->event.abi), &child->inj.reg);
         ^
In file included from tracy-modification.c:23:0:
tracy.h:459:5: note: expected ‘struct tracy_sc_args *’ but argument is of type ‘struct pt_regs *’
 int tracy_modify_syscall_args(struct tracy_child *child, long syscall_number,
     ^
tracy-modification.c:135:9: note: #pragma message: Verify that this abi is ok; do we want to pass it instead?
 #pragma message "Verify that this abi is ok; do we want to pass it instead?"
         ^
tracy-modification.c: In function ‘tracy_inject_syscall_post_start’:
tracy-modification.c:227:9: note: #pragma message: Verify that this abi is ok; do we want to pass it instead?
 #pragma message "Verify that this abi is ok; do we want to pass it instead?"
         ^
tracy-modification.c: In function ‘tracy_modify_syscall_args’:
tracy-modification.c:302:9: note: #pragma message: Verify that this abi is ok; do we want to pass it instead?
 #pragma message "Verify that this abi is ok; do we want to pass it instead?"
         ^
cc1: all warnings being treated as errors
<builtin>: recipe for target 'tracy-modification.o' failed
make: *** [tracy-modification.o] Error 1